packages/salesforcedx-utils-vscode/src/settings/index.ts

```diff
@@ -1 +1,6 @@
+import { SfdxCoreSettings } from './sfdxCoreSettings';
+
 export * from './sfdxSettingsService';
+export * from './sfdxCoreSettings';
+
+export const sfdxCoreSettings = SfdxCoreSettings.getInstance();
```

Comment:
FYI I know we do it in this repo some but exporting instances lead to sadness often when trying to test the code (b/c anything that loads this file will also instantiates the underlying instance)

Here can we export a method to get the sfdxCoreSettings and just use that getting method...along the lines of 
```
let sfdxCoreSettingsInst;
export const getSfdxCoreSettings = () => {
  if (!sfdxCoreSetttingsInst) {
     sfdxCoreSettings = SfdxCoreSettings.getInstance();
  }
  return sfdxCoreSettingsInst;
}
```